packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js

```diff
@@ -31,6 +31,23 @@ export default /* @ngInject */ ($stateProvider) => {
 
         breadcrumb: /* @ngInject */ $translate => $translate.instant('pci_projects_project_network_private'),
 
+        goToPrivateNetworks: /* @ngInject */ (CucCloudMessage, $state, projectId) => (message = false, type = 'success') => {
```

Comment:
Could you please sort parameters (and group by `$*` first) in order to be consistent?

Reply:
(Keep `projectId` at the end as it results from `resolve`)